### PR TITLE
Fix/various

### PR DESCRIPTION
**Pull request by boundlesscalm**

### CODE REVIEW

**Comment by KievDevel** [comment]
Probably should add changelog entry as well?

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 0.2.6 - 2024-04-12
+
+### Changed
+
+-   Updated Nordic Semiconductor documentation links.
+
+### Fixed
+
+-   A skip button is now shown in the verification step if verification fails.
```
Comment (greg-fer) [inline Changelog.md:9]: Shouldn't this be a ### Changed or ### Added?

Comment (greg-fer) [inline Changelog.md:9]:
```suggestion
-   Skip button is now shown in the verification step if the verification fails.
```

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+-   Modem firmware version v2.0.1 was mistaknly displayed as v2.0.0 for nRF91x1
+    devices.
```
Comment (greg-fer) [inline Changelog.md:11]:
```suggestion
-   Modem firmware version v2.0.1 was wrongly displayed as v2.0.0 for nRF9151 and nRF9161
    devices.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 1 above)
Comment (boundlesscalm) [reply Changelog.md:9]: There was a skip button before but it was shown during weird times only.
I get why my wording would give that impression though.

File: Changelog.md (hunk 2 above)
Comment (boundlesscalm) [reply Changelog.md]: I changed it to say "nRF9151 DK and nRF9161 DK", I hope that's fine.

**Review by greg-fer — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (greg-fer) [reply Changelog.md]: Add "again" between "now" and "shown"?

**Pull request merged**